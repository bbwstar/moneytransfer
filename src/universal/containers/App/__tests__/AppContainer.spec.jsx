import React from 'react';
import renderer from 'react-test-renderer';

import AppContainer from '../AppContainer';


jest.mock('components/App/App', () => 'App');

describe('AppContainer (Container)', () => {
  it('renders without exploding', () => {
    const component = renderer.create(
      <AppContainer>
        <div className="test" />
      </AppContainer>,
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
