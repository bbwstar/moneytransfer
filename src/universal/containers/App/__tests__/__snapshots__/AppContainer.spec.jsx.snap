// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer (Container) renders without exploding 1`] = `
<App>
  <div
    className="test"
  />
</App>
`;
